project/settings.py

```diff
@@ -55,6 +55,8 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'django.middleware.security.SecurityMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```

Comment:
The [docs](http://whitenoise.evans.io/en/stable/django.html#enable-whitenoise) say to place it after `django.middleware.security.SecurityMiddleware`, so this looks good :+1: